examples/pipelines/mnist_vertical/submit.py

```diff
@@ -0,0 +1,317 @@
+
+    # Get a handle to workspace
+    try:
+        # tries to connect using cli args if provided else using config.yaml
+        ML_CLIENT = MLClient(
+            subscription_id=args.subscription_id or YAML_CONFIG.aml.subscription_id,
+            resource_group_name=args.resource_group
+            or YAML_CONFIG.aml.resource_group_name,
+            workspace_name=args.workspace_name or YAML_CONFIG.aml.workspace_name,
+            credential=credential,
+        )
+
+    except Exception as ex:
+        print("Could not find either cli args or config.yaml.")
+        # tries to connect using local config.json
+        ML_CLIENT = MLClient.from_config(credential=credential)
+
+    return ML_CLIENT
+
+
+####################################
+### LOAD THE PIPELINE COMPONENTS ###
+####################################
+
+# Loading the component from their yaml specifications
+training_contributor_component = load_component(
+    source=os.path.join(COMPONENTS_FOLDER, "traininsilo", "contributor_spec.yaml")
+)
+
+training_host_component = load_component(
+    source=os.path.join(COMPONENTS_FOLDER, "traininsilo", "host_spec.yaml")
+)
+
+
+########################
+### BUILD A PIPELINE ###
+########################
+
+
+def custom_fl_data_path(
+    datastore_name, output_name, unique_id="${{name}}", iteration_num=None
+):
+    """Produces a path to store the data during FL training.
+
+    Args:
+        datastore_name (str): name of the Azure ML datastore
+        output_name (str): a name unique to this output
+        unique_id (str): a unique id for the run (default: inject run id with ${{name}})
+        iteration_num (str): an iteration number if relevant
+
+    Returns:
+        data_path (str): direct url to the data path to store the data
+    """
+    data_path = f"azureml://datastores/{datastore_name}/paths/federated_learning/{output_name}/{unique_id}/"
+    if iteration_num:
+        data_path += f"iteration_{iteration_num}/"
+
+    return data_path
+
+
+def getUniqueIdentifier(length=8):
+    """Generates a random string and concatenates it with today's date
+
+    Args:
+        length (int): length of the random string (default: 8)
+
+    """
+    str = string.ascii_lowercase
+    date = datetime.date.today().strftime("%Y_%m_%d_")
+    return date + "".join(random.choice(str) for i in range(length))
+
+
+pipeline_identifier = getUniqueIdentifier()
+
+
+@pipeline(
+    description=f'FL cross-silo basic pipeline and the unique identifier is "{pipeline_identifier}" that can help you to track files in the storage account.',
+)
+def fl_mnist_vertical_basic():
+    ################
+    ### TRAINING ###
+    ################
+
+    outputs = {}
+    # for each silo, run a distinct training with its own inputs and outputs
+    for silo_index, silo_config in enumerate(
+        [YAML_CONFIG.federated_learning.host] + YAML_CONFIG.federated_learning.silos
+    ):
+        if silo_index == 0:
```

Comment:
Is it mentioned anywhere that the silo order is important now?

Reply:
`[YAML_CONFIG.federated_learning.host] + YAML_CONFIG.federated_learning.silos` this makes sure that host is always the first item to iterate over